reanahub/reana-workflow-controller
# rest: include source as part of workflows endpoint

closes reanahub/reana-ui#222

**To test:**

Hardcode different sources and check result in [the UI](https://github.com/reanahub/reana-ui/pull/228):

```diff
﻿﻿diff --git a/reana_workflow_controller/rest/workflows.py b/reana_workflow_controller/rest/workflows.py
index 1c29805..3c4fe44 100644
--- a/reana_workflow_controller/rest/workflows.py
+++ b/reana_workflow_controller/rest/workflows.py
@@ -258,7 +258,10 @@ def get_workflows(paginate=None):  # noqa
                 "name": get_workflow_name(workflow),
                 "status": workflow.status.name,
                 "user": user_uuid,
-                "source": workflow.source,
+                # "source": workflow.source,
+                "source": "https://github.com/reanahub/reana-demo-helloworld",
+                # "source": "https://zenodo.org/record/1/reana.yaml",
+                # "source": "https://mycdn.org/workflow-specs/reana-snakemake.yaml",
                 "created": workflow.created.strftime(WORKFLOW_TIME_FORMAT),
                 "progress": get_workflow_progress(
                     workflow, include_progress=include_progress
```
